README.md

```diff
@@ -1 +1,22 @@
-# hw01
\ No newline at end of file
+##### HOMEWORK ASSIGNMENT 1
```

Comment:
For a top-level header in a document, I suggest a level 1 or 2 header. Notice the difference in font size and style

```
# Homework assignment 1
## Homework assignment 1
### Homework assignment 1
#### Homework assignment 1
##### Homework assignment 1
```

# Homework assignment 1
## Homework assignment 1
### Homework assignment 1
#### Homework assignment 1
##### Homework assignment 1